src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,33 @@
 import React from 'react';
+import { Comments } from '../../types/Comments';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  comment: Comments;
+};
+
+export const CommentInfo: React.FC<Props> = ({
+  comment: {
+    name,
+    email,
+    body,
+  },
```

Comment:
It's better to use destructuring inside the body of the function, not in arguments directly